modules/80_optimization/nash/postsolve.gms

```diff
@@ -659,28 +659,37 @@ if (cm_abortOnConsecFail gt 0,
   );
 
   if (smax(regi, p80_trackConsecFail(regi)) >= cm_abortOnConsecFail,
-    execute_unload "abort.gdx";
-    display p80_trackConsecFail;
-    abort "Run was aborted because the maximum number of consecutive failures was reached in at least one region!";
+    if (s80_runInDebug eq 0,
```

Comment:
I think this could use a further case for the control flow.
Sometimes the infeasibility of one region propagates to all other regions, presumably via the trade prices, which leads to all regions (see example below).
I think in such a case it is pointless to run all regions in debug mode, since no information will be gleaned from that.
Should be testable by `sum(regi, p80_SolNonInfes(regi) ne 0) eq 0`.

```
$ nashstat -a /p/projects/remind/modeltests/remind/output/SSP2EU_PBS-NPi-cali
brate-AMT_2024-03-15_22.10.09/ | sed -n '/^ 49/,$ p'
 49 	solvestat            	modelstat               	resusd   	objval           
CAZ 	terminated by solver 	      feasible solution 	 234.111 	 7.13556120105577 	
CHA 	   normal completion 	        locally optimal 	 319.421 	 60.4582498064749 	
EUR 	   normal completion 	        locally optimal 	 842.578 	 40.7912528898278 	
IND 	terminated by solver 	      feasible solution 	 710.385 	 3.20429604137173 	
JPN 	   normal completion 	        locally optimal 	 139.562 	 8.08978449648259 	
LAM 	terminated by solver 	      feasible solution 	  57.276 	  45.298029542137 	
MEA 	terminated by solver 	      feasible solution 	 291.212 	 46.1969898464668 	
NEU 	   normal completion 	        locally optimal 	 114.667 	 9.19402360457552 	
OAS 	   normal completion 	        locally optimal 	 252.736 	 67.3169961522613 	
REF 	terminated by solver 	      feasible solution 	 113.433 	 18.1272936313763 	
SSA 	   normal completion 	        locally optimal 	 385.797 	 95.9642805867113 	
USA 	   normal completion 	        locally optimal 	 124.476 	 32.9174606296399 	

 50 	solvestat            	modelstat               	resusd   	objval           
CAZ 	   normal completion 	        locally optimal 	 203.165 	 7.13530668263627 	
CHA 	terminated by solver 	      feasible solution 	1957.261 	-324.789411300748 	
EUR 	   normal completion 	        locally optimal 	 383.149 	 40.6820863108663 	
IND 	   NORMAL COMPLETION 	     LOCALLY INFEASIBLE 	 451.969 	 -53471.643251211 	F
JPN 	   normal completion 	        locally optimal 	 263.363 	 8.05553055211674 	
LAM 	terminated by solver 	      feasible solution 	 490.307 	  45.852113704894 	
MEA 	   NORMAL COMPLETION 	     LOCALLY INFEASIBLE 	 283.948 	-6573.71436599867 	F
NEU 	terminated by solver 	      feasible solution 	 144.295 	 9.21957927102501 	
OAS 	TERMINATED BY SOLVER 	INTERMEDIATE INFEASIBLE 	3029.602 	 -160092.42893471 	F
REF 	   NORMAL COMPLETION 	     LOCALLY INFEASIBLE 	 487.129 	-2123.04334911169 	F
SSA 	terminated by solver 	      feasible solution 	 500.664 	 96.4200069039671 	
USA 	   normal completion 	        locally optimal 	   182.7 	 32.8749092042601 	

 51 	solvestat            	modelstat               	resusd   	objval           
CAZ 	   normal completion 	        locally optimal 	  229.83 	 7.14319954466072 	
CHA 	   NORMAL COMPLETION 	     LOCALLY INFEASIBLE 	2083.307 	-114821.914533384 	F
EUR 	TERMINATED BY SOLVER 	INTERMEDIATE INFEASIBLE 	  759.42 	-17028.4982609693 	F
IND 	   NORMAL COMPLETION 	     LOCALLY INFEASIBLE 	1842.799 	-179293.779033935 	F
JPN 	   NORMAL COMPLETION 	     LOCALLY INFEASIBLE 	 1263.89 	-741.694383966832 	F
LAM 	TERMINATED BY SOLVER 	INTERMEDIATE INFEASIBLE 	1953.653 	 -35419.065349658 	F
MEA 	TERMINATED BY SOLVER 	INTERMEDIATE INFEASIBLE 	 669.345 	-37904.1121634802 	F
NEU 	TERMINATED BY SOLVER 	INTERMEDIATE INFEASIBLE 	1648.559 	 -1075.1809233486 	F
OAS 	   NORMAL COMPLETION 	     LOCALLY INFEASIBLE 	2651.263 	-160092.839313142 	F
REF 	   NORMAL COMPLETION 	     LOCALLY INFEASIBLE 	2137.533 	-6483.10571837927 	F
SSA 	TERMINATED BY SOLVER 	INTERMEDIATE INFEASIBLE 	1766.617 	-338469.926267192 	F
USA 	   NORMAL COMPLETION 	     LOCALLY INFEASIBLE 	 1594.47 	-14059.7982834075 	F

 52 	solvestat            	modelstat               	resusd   	objval           
CAZ 	TERMINATED BY SOLVER 	INTERMEDIATE INFEASIBLE 	1442.223 	-676.840960843086 	F
CHA 	   NORMAL COMPLETION 	     LOCALLY INFEASIBLE 	 997.374 	-114822.007952252 	F
EUR 	TERMINATED BY SOLVER 	INTERMEDIATE INFEASIBLE 	  770.76 	-18258.0128586332 	F
IND 	   NORMAL COMPLETION 	     LOCALLY INFEASIBLE 	1135.956 	-179294.100231812 	F
JPN 	TERMINATED BY SOLVER 	INTERMEDIATE INFEASIBLE 	2262.399 	-823.367837768392 	F
LAM 	TERMINATED BY SOLVER 	INTERMEDIATE INFEASIBLE 	 603.014 	-36261.0143771092 	F
MEA 	TERMINATED BY SOLVER 	INTERMEDIATE INFEASIBLE 	 293.088 	-37904.1111966965 	F
NEU 	   NORMAL COMPLETION 	     LOCALLY INFEASIBLE 	  481.85 	-1125.42404218371 	F
OAS 	TERMINATED BY SOLVER 	INTERMEDIATE INFEASIBLE 	 499.509 	-160092.782483673 	F
REF 	   NORMAL COMPLETION 	     LOCALLY INFEASIBLE 	 805.382 	-6489.40554589407 	F
SSA 	   NORMAL COMPLETION 	     LOCALLY INFEASIBLE 	 421.309 	-338470.071882028 	F
USA 	   NORMAL COMPLETION 	     LOCALLY INFEASIBLE 	 453.817 	-14059.8224474974 	F

 53 	solvestat            	modelstat               	resusd   	objval           
CAZ 	   NORMAL COMPLETION 	     LOCALLY INFEASIBLE 	  54.864 	-676.849916840935 	F
CHA 	EVALUATION INTERRUPT 	INTERMEDIATE INFEASIBLE 	    0.06 	-14059.6328933037 	F
EUR 	EVALUATION INTERRUPT 	INTERMEDIATE INFEASIBLE 	   0.058 	-14059.6328933037 	F
IND 	   NORMAL COMPLETION 	     LOCALLY INFEASIBLE 	 507.931 	-179294.027260981 	F
JPN 	TERMINATED BY SOLVER 	INTERMEDIATE INFEASIBLE 	 106.492 	-823.378262113495 	F
LAM 	EVALUATION INTERRUPT 	INTERMEDIATE INFEASIBLE 	    0.06 	-14059.6328933037 	F
MEA 	EVALUATION INTERRUPT 	INTERMEDIATE INFEASIBLE 	   0.053 	-14059.6328933037 	F
NEU 	EVALUATION INTERRUPT 	INTERMEDIATE INFEASIBLE 	   0.055 	-14059.6328933037 	F
OAS 	EVALUATION INTERRUPT 	INTERMEDIATE INFEASIBLE 	   0.058 	-14059.6328933037 	F
REF 	EVALUATION INTERRUPT 	INTERMEDIATE INFEASIBLE 	   0.056 	-14059.6328933037 	F
SSA 	EVALUATION INTERRUPT 	INTERMEDIATE INFEASIBLE 	   0.053 	-14059.6328933037 	F
USA 	TERMINATED BY SOLVER 	INTERMEDIATE INFEASIBLE 	  519.35 	-14059.7648229642 	F

 54 	solvestat            	modelstat               	resusd   	objval           
CAZ 	EVALUATION INTERRUPT 	INTERMEDIATE INFEASIBLE 	   0.055 	-14059.5405476234 	F
CHA 	EVALUATION INTERRUPT 	INTERMEDIATE INFEASIBLE 	   0.055 	-14059.5405476234 	F
EUR 	EVALUATION INTERRUPT 	INTERMEDIATE INFEASIBLE 	   0.056 	-14059.5405476234 	F
IND 	EVALUATION INTERRUPT 	INTERMEDIATE INFEASIBLE 	   0.056 	-14059.5405476234 	F
JPN 	EVALUATION INTERRUPT 	INTERMEDIATE INFEASIBLE 	   0.054 	-14059.5405476234 	F
LAM 	EVALUATION INTERRUPT 	INTERMEDIATE INFEASIBLE 	    0.06 	-14059.5405476234 	F
MEA 	EVALUATION INTERRUPT 	INTERMEDIATE INFEASIBLE 	   0.056 	-14059.5405476234 	F
NEU 	EVALUATION INTERRUPT 	INTERMEDIATE INFEASIBLE 	   0.054 	-14059.5405476234 	F
OAS 	EVALUATION INTERRUPT 	INTERMEDIATE INFEASIBLE 	   0.063 	-14059.5405476234 	F
REF 	EVALUATION INTERRUPT 	INTERMEDIATE INFEASIBLE 	   0.055 	-14059.5405476234 	F
SSA 	EVALUATION INTERRUPT 	INTERMEDIATE INFEASIBLE 	   0.054 	-14059.5405476234 	F
USA 	   NORMAL COMPLETION 	     LOCALLY INFEASIBLE 	 165.159 	-14059.4963092159 	F
```

Reply:
Added this condition